docs/DeveloperGuide.md

```diff
@@ -316,14 +316,14 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
       Use case resumes at step 1.
 
 
-#### Use case: Delete a stock
+#### Use case: Deleting stocks
```

Comment:
Maybe can include the word "multiple" to make the use case more explicit? Some people with poor English like me might miss this out.